mgoroshko/h:w2_Tables_Links/python_3_w_links.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<META charset="utf-8">
+<title>HOW-TO: Программа на Python, Часть 3</title>
+</head>
+
+<body>
+<h1>HOW-TO: Программа на Python, Часть 3</h1>
+<ul>
+        <em>
+<li><a href="python_1_w_links.html">Программа на Python — часть 1</a></li>
+<li><a href="python_2_w_links.html">Программа на Python — часть 2</a></li>
+<li>Программа на Python — часть 3</li>
+</em>
+</ul>
+<h2>Содержание</h2>
+<h3>HOW-TO: Программа на Python, Часть 3</h3>
+<ul><u>
```

Comment:
Inline tag should be inside each li

Reply:
fixed everything, can you see the commits?